### PR TITLE
Static wasm resolver

### PR DESCRIPTION
**Pull request by ramnivas**

### CODE REVIEW

File: crates/wasm-subsystem/wasm-resolver-dynamic/src/lib.rs [hunk 1]
```diff
@@ -7,8 +7,11 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-pub use plugin::WasmSubsystemLoader;
+//! Dynamic loader for postgres-resolver.
```
Comment (shadaj) [inline crates/wasm-subsystem/wasm-resolver-dynamic/src/lib.rs:10]: `wasm-resolver`

File: crates/wasm-subsystem/wasm-resolver-dynamic/src/lib.rs [hunk 2]
```diff
@@ -7,8 +7,11 @@
...
 
-mod plugin;
-mod wasm_execution_error;
-mod wasm_operation;
+use core_plugin_interface::interface::SubsystemLoader;
+use wasm_resolver::WasmSubsystemLoader;
+
+// See comments in `postgres-resolver-dynamic/src/lib.rs`.
```
Comment (shadaj) [inline crates/wasm-subsystem/wasm-resolver-dynamic/src/lib.rs:15]: and here

File: crates/wasm-subsystem/wasm-resolver-dynamic/src/lib.rs (hunk 1 above)
Comment (ramnivas) [reply crates/wasm-subsystem/wasm-resolver-dynamic/src/lib.rs:10]: Good catch. Fixed.

File: crates/wasm-subsystem/wasm-resolver-dynamic/src/lib.rs (hunk 2 above)
Comment (ramnivas) [reply crates/wasm-subsystem/wasm-resolver-dynamic/src/lib.rs:15]: This one is okay. The comment is in postgres-resolver-dynamic.

**Review by shadaj — APPROVED** [review]

**Pull request merged**